### PR TITLE
Add new loopback rule

### PR DESCRIPTION
**Pull request by anivan-suse**
#### Description:

Add new rule to PCI-DSS
- set_loopback_traffic
- set_ipv6_loopback_traffic

### CODE REVIEW

File: linux_os/guide/system/network/network-iptables/iptables_activation/set_loopback_traffic/rule.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    cce@sle15: CCE-91345-9
+
+references:
+    cis@sle15: 3.5.2.6
+    pcidss: Req-1.4.1
+
+warnings:
+    - general: |-
+    Changing firewall settings while connected over network can 
+    result in being locked out of the system.
```
Comment (ggbecker) [inline linux_os/guide/system/network/network-iptables/iptables_activation/set_loopback_traffic/rule.yml:30]:
```suggestion
        Changing firewall settings while connected over network can 
        result in being locked out of the system.
```

File: controls/cis_sle_15.yml [hunk 2]
```diff
@@ -1012,6 +1012,8 @@ controls:
       - l1_server
       - l1_workstation
     status: manual
+    rule:
```
Comment (jan-cerny) [inline controls/cis_sle_15.yml:1015]: This should be `rules:`.

**Review by jan-cerny — CHANGES_REQUESTED** [review]

File: controls/cis_sle_15.yml (hunk 2 above)
Comment (jan-cerny) [inline controls/cis_sle_15.yml:1015]: There should be `rules`, not `rule`.

**Review by jan-cerny — CHANGES_REQUESTED** [review]

**Comment by jan-cerny** [comment]
The CI fail in Fedora rawhide is unrelated to this PR, it's caused by libxml2 ABI break, see https://bugzilla.redhat.com/show_bug.cgi?id=2139546

**Pull request merged**